site/topics/lists/alias-trivia.rst

```diff
@@ -140,143 +140,94 @@ Side Effects & Pure Functions
     * Sometimes it's just a lot easier to achieve something with side effects
 
 
-Nested lists
```

Comment:
bad diff -- checked out from the wrong branch it looks like

Reply:
Bingo Bango